code/modules/events/brain_trauma.dm

```diff
@@ -0,0 +1,35 @@
+/datum/round_event_control/brain_trauma
+	name = "Spontaneous Brain Trauma"
+	typepath = /datum/round_event/brain_trauma
+	weight = 25
+
+/datum/round_event/brain_trauma
+	fakeable = FALSE
+
+/datum/round_event/brain_trauma/start()
+	for(var/mob/living/carbon/human/H in shuffle(GLOB.alive_mob_list))
+		if(!H.client)
+			continue
+		if(H.stat == DEAD) // What are you doing in this list
+			continue
+		if(!H.getorgan(/obj/item/organ/brain)) // If only I had a brain
+			continue
+
+		traumatize(H)
+		announce_to_ghosts(H)
+		break
+
+/datum/round_event/brain_trauma/proc/traumatize(mob/living/carbon/human/H)
+	var/resistance = pick(
+		50;TRAUMA_RESILIENCE_BASIC,
+		30;TRAUMA_RESILIENCE_SURGERY,
+		15;TRAUMA_RESILIENCE_LOBOTOMY,
+		5;TRAUMA_RESILIENCE_MAGIC)
```

Comment:
have you considered removing the magic resilience because this is being phased out of various things, and just isn't fun to have

Reply:
Someone already ported the event without lobotomy/magic resilience bits it seems. :shrug: